tests/test_data.py

```diff
@@ -7,28 +8,34 @@
 cwd = os.path.abspath(os.path.dirname(__file__))
 test_data_dir = os.path.join(cwd, "data")
 test_out_dir = os.path.join(cwd, "tmp")
+os.makedirs(test_out_dir, exist_ok=True)
 test_validate_dir = os.path.join(cwd, "validate_data")
 schema_dir = os.path.join(cwd, "../schema")
 TEST_CONFIG = os.path.join(cwd, "test_config.yaml")
 DEFAULT_CONTEXT_PATH = os.path.join(schema_dir, "sssom.context.jsonld")
 
 
-def get_test_file(filename):
+def get_test_file(filename: str) -> str:
+    """Get a test file path inside the test data directory."""
     return os.path.join(test_data_dir, filename)
 
 
-def ensure_test_dir_exists():
-    if not os.path.exists(test_out_dir):
-        os.makedirs(test_out_dir)
-
-
 def load_config():
+    """Load test file information from 'test_config.yaml'.
+
+    :return: Confiuration
```

Comment:
Typo. Also, if the return is redundant of the one line explanation at the top, you might as well skip it